app/src/main/java/org/kiwix/kiwixmobile/utils/SharedPreferenceUtil.java

```diff
@@ -172,4 +173,12 @@ public void setShowBookmarksCurrentBook(boolean prefShowBookmarksFromCurrentBook
         .putBoolean(PREF_SHOW_BOOKMARKS_CURRENT_BOOK, prefShowBookmarksFromCurrentBook)
         .apply();
   }
+
+  public void updateLastBookOpened(String title){
```

Comment:
Actually, I used the title because bookname is null for most of the ZIM files, but sorry I didn't realize that title is common for multiple files. I will change it and use the id of the files.